docs/migration_quickstart.md

```diff
@@ -0,0 +1,295 @@
+
+    ```sh
+    kubectl -n crossplane-system scale --replicas=1 deployment/crossplane-rbac-manager
+    kubectl -n crossplane-system scale --replicas=1 deployment/crossplane
+    ```
+    Verify:
+
+    ```sh
+    kubectl get -n crossplane-system deployment
+    NAME                                                READY   UP-TO-DATE   AVAILABLE   AGE
+    crossplane                                          1/1     1            1           6d23h
+    crossplane-rbac-manager                             1/1     1            1           6d23h
+    oracle-samples-provider-family-oci-3f6aefd6de9e     1/1     1            1           38h
+    oracle-samples-provider-oci-objectstorage-8c4d476   1/1     1            1           38h
+    ```
+
+---
+
+### 2. Migration: Use a New Family Provider Name, but Keep Sub Provider Names the Same (Required for Sub Providers)
+
+You may assign a new metadata name to the family provider, but you **must** keep the same sub provider names.
+
+#### Steps:
+
+1. **Check the current providers and note the metadata names (e.g., `oracle-samples-provider-family-oci`, `oracle-samples-provider-oci-objectstorage`):**
+
+    ```sh
+    kubectl get providers
+    NAME                                       INSTALLED   HEALTHY   PACKAGE                                                                  AGE
+    oracle-samples-provider-family-oci          True        True      ghcr.io/oracle-samples/provider-family-oci:v0.0.1-alpha.1-amd64          3m3s
+    oracle-samples-provider-oci-objectstorage   True        True      ghcr.io/oracle-samples/provider-oci-objectstorage:v0.0.1-alpha.1-amd64   3m2s
+    ```
+
+2. **Pause Crossplane and RBAC Manager:**
+
+   > This prevents the controllers from reconciling resources during provider migration, which could lead to resource deletion or orphaning.
+
+   ```sh
+    kubectl -n crossplane-system scale --replicas=0 deployment/crossplane-rbac-manager
+    kubectl -n crossplane-system scale --replicas=0 deployment/crossplane
+    ```
+    Verify:
+
+    ```sh
+    kubectl get -n crossplane-system deployment
+    NAME                                                       READY   UP-TO-DATE   AVAILABLE   AGE
+    crossplane                                                 0/0     0            0           6d23h
+    crossplane-rbac-manager                                    0/0     0            0           6d23h
+    oracle-samples-provider-family-oci-3f6aefd6de9e             1/1     1            1           38h
+    oracle-samples-provider-oci-objectstorage-8c4d47602759      1/1     1            1           38h
+    ```
+
+4. **Delete ProviderConfig (force remove finalizers if needed):**
+
+  First, attempt a normal deletion:
+  ```sh
+  kubectl delete providerconfig/default
+  ```
+
+  If the deletion hangs due to finalizers, you can force removal in a separate command:
+
+  > ⚠️ **WARNING**: Force-removing finalizers bypasses Kubernetes safety mechanisms.
```

Comment:
```suggestion
  > [!WARNING] 
  > Force-removing finalizers bypasses Kubernetes safety mechanisms.
```

This is from the alerts markdown https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts